### PR TITLE
Changed sizing on filters

### PR DESCRIPTION
**Pull request by agonza40**
Changed styling to decrease the size of the filters and better align them with the info-bar.

#PATCH#

### CODE REVIEW

**Comment by cstolli** [comment]
:+1:

File: addon/styles/addon.scss [hunk 1]
```diff
@@ -3,9 +3,9 @@
 .frost-object-browser {
   display: flex;
   flex-direction: column;
-  padding: 0 30px;
 
   .frost-info-bar {
+    padding: 0 30px;
```
Comment (sglanzer) [inline addon/styles/addon.scss]: Info bar styles should be changed globally as part of https://github.com/ciena-frost/ember-frost-info-bar/blob/master/addon/styles/addon.scss

**Comment by sglanzer** [comment]
Rejected, please address the info bar style prior to merge

File: addon/styles/addon.scss (hunk 1 above)
Comment (agonza40) [inline addon/styles/addon.scss]: @sglanzer Can you confirm that these styles are desired globally? (`.frost-info-bar` and `.frost-info-bar .title`)

**Comment by agonza40** [comment]
@sglanzer I've addressed the info bar styling. Turns out it was originally needed to correct other styling that has been removed.

**Comment by sglanzer** [comment]
Approved - thanks @agonza40

**Pull request merged**